### PR TITLE
Gin n Tonic Kotlin front page  ADD

### PR DESCRIPTION
**Pull request by HakanEgeOzerol**

### CODE REVIEW

**Review by JoeyKraemer — DISMISSED** [review]
Please review your code for code conventions,  in a lot of different files there is the empty line at the end missing

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,122 +4,149 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/tonic_text"
+        app:srcCompat="@drawable/gin" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="441dp"
+        android:layout_height="436dp"
+        android:layout_marginStart="109dp"
+        android:layout_marginTop="190dp"
+        android:text="Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/activity_main.xml:134]: tab the text in

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,122 +4,149 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="343dp"
+        android:layout_marginEnd="334dp"
+        android:layout_marginBottom="3dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/imageView3"
+        app:layout_constraintStart_toEndOf="@+id/imageView6"
+        app:srcCompat="@drawable/order" />
+
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/activity_main.xml:150]: empty lines to many

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_rum_coke.xml [hunk 3]
```diff
@@ -4,6 +4,149 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/ligthBeige"
     tools:context=".Rum_coke">
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/activity_rum_coke.xml:9]: empty lines 2 you usually do 1

File: app/src/main/res/layout/activity_rum_coke.xml [hunk 4]
```diff
@@ -4,6 +4,149 @@
...
+
+        android:layout_marginTop="1dp"
+        android:layout_marginEnd="1066dp"
+        android:layout_marginBottom="13dp"
+        app:layout_constraintBottom_toBottomOf="@+id/title"
+        app:layout_constraintEnd_toStartOf="@+id/imageView4"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/mint" />
+
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/activity_rum_coke.xml:42]: here there are 3

File: app/src/main/res/layout/activity_rum_coke.xml [hunk 5]
```diff
@@ -4,6 +4,149 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/title"
+        app:srcCompat="@drawable/mocktail" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="493dp"
+        android:layout_height="403dp"
+        android:layout_marginStart="109dp"
+        android:layout_marginTop="190dp"
+        android:text="Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/activity_rum_coke.xml:121]: tab text in

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

**Review by AlvesBernardo — APPROVED** [review]

**Pull request merged**